lib/arclight/shared_indexing_behavior.rb

```diff
@@ -5,14 +5,33 @@ module Arclight
   # A mixin intended to share indexing behavior between
   # the CustomDocument and CustomComponent classes
   module SharedIndexingBehavior
+    # TODO: probably should have a proper DateRange class at some point
+    def to_year_from_iso8601(date)
+      return nil if date.blank?
```

Comment:
`return nil` is a bit redundant here.  As in the `to_date_range` method we can just `return`.  I have a slight preference towards not explicitly returning `nil` and just returning, but I think we should be consistent between the two methods.